Dockerfile

```diff
@@ -1,10 +1,6 @@
 FROM ghcr.io/xmtp/rust:latest
 USER xmtp
 
-RUN ~xmtp/.cargo/bin/rustup toolchain install stable 
```

Comment:
These components are already in the base image, so we don't need to install them again. For some reason the re-install is failing.

Reply:
Seems related to [this issue](https://github.com/rust-lang/rustup/issues/1239) (same error). Dockerignoring the `rust-toolchain.toml` file does avoid the issue, at the cost of forcing us to use the rust toolchain from the base docker file instead of the one specified in the `rust-toolchain.toml`

Reply:
@jac18281828 might be able to help, maybe because of some updates?